dataikuapi/dss/recipe.py

```diff
@@ -1376,6 +1388,66 @@ def with_output_evaluation_store(self, mes_id):
         return self._with_output(mes_id, role="evaluationStore")
 
 
+class StandaloneEvaluationRecipeCreator(DSSRecipeCreator):
+    """
+    Builder for the creation of a new "Standalone Evaluate" recipe, from an
+    input dataset
+
+    .. code-block:: python
+
+        # Create a new standalone evaluation of a dataset
+
+        project = client.get_project("MYPROJECT")
+        builder = StandaloneEvaluationRecipeCreator("my_standalone_evaluation_recipe", project)
+        builder.with_input("scored_dataset_to_evaluate")
+        builder.with_output_evaluation_store(evaluation_store_id)
+
+        new_recipe = builder.build()
+
+        # Save the model parameters in the SER settings and run the SER
+
+        ser_payload = dict(predictionType="BINARY_CLASSIFICATION",
+                   targetVariable="Survived",
+                   predictionVariable="prediction",
+                   isProbaAware=True,
+                   dontComputePerformance=False)
+
+        class_0 = dict(key=0, value="proba_0")
+        class_1 = dict(key=1, value="proba_1")
+        ser_payload['probas'] = [class_0, class_1]
+
+        feature_passengerid = dict(name="Passenger_Id", role="REJECT", type="TEXT")
+        feature_ticket = dict(name="Ticket", role="REJECT", type="TEXT")
+        feature_cabin = dict(name="Cabin", role="REJECT", type="TEXT")
+
+        ser_payload['features'] = [feature_passengerid, feature_ticket, feature_cabin]
+
+        ser_settings = ser.get_settings()
+
+        # Add the newly created json payload to the recipe settings
+        # Note that with this method, all the settings that were not explicitly set are instead set to their default value.
+        # e.g. there is an empty cost matrix setting for this recipe because 'cost matrix' is not defined in the payload
```

Comment:
So, I would give an example in this doc of setting the costMatrix to the default values.